etsin_finder/frontend/js/components/qvain/files/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Translate from 'react-translate-component'
+import { SectionTitle } from '../general/section'
+import { ContainerLight, ContainerSubsection } from '../general/card'
+import IDAFilePicker from './idaFilePicker'
+import ExternalFiles from './externalFiles'
+
+class Files extends Component {
```

Comment:
Highest level component for file section of Qvain Light form. This is the parent component for the entire section (includes 2 cards).